src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx

```diff
@@ -666,6 +666,25 @@ Use [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clien
       </thead>
 
       <tbody>
+        <tr>
+          <td>
+            Confluent.Kafka
+          </td>
+
+          <td>
+            * Produce and Consume on Topics.
```

Comment:
@tippmar-nr Can you clarify what this bullet point means?

Reply:
These are the type of actions ("produce" and "consume") we instrument in Kafka.